nix/cardano-services/deployments/blockfrost-backend.provider.nix

```diff
@@ -0,0 +1,97 @@
+{
+  config,
+  values,
+  lib,
+  ...
+}: {
+  providers.blockfrost-backend = {
+    inherit (values.cardano-services) image;
+    args = ["start-provider-server"];
+    port = 3000;
+    metricsPath = "${values.cardano-services.httpPrefix}/metrics";
+
+    livenessProbe = {
+      timeoutSeconds = 30;
+      periodSeconds = 60;
+      httpGet = {
+        path = "${values.cardano-services.httpPrefix}/health";
+        port = 3000;
+      };
+    };
+
+    env =
+      {
+        NETWORK = config.network;
+        ENABLE_METRICS = "true";
+        SERVICE_NAMES = "asset,chain-history,network-info,rewards,utxo";
+        OGMIOS_SRV_SERVICE_NAME = values.backend.ogmiosSrvServiceName;
+        LOGGER_MIN_SEVERITY = values.cardano-services.loggingLevel;
+        TOKEN_METADATA_SERVER_URL = values.cardano-services.tokenMetadataServerUrl;
+        HANDLE_POLICY_IDS = "f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a";
+        USE_BLOCKFROST = builtins.toJSON values.blockfrost-worker.enabled;
+        USE_KORA_LABS = "true";
+        DISABLE_STAKE_POOL_METRIC_APY = "true";
+        PAGINATION_PAGE_SIZE_LIMIT = "5500";
+        NODE_ENV = values.cardano-services.nodeEnv;
+        ASSET_PROVIDER = "blockfrost";
+        CHAIN_HISTORY_PROVIDER = "blockfrost";
+        NETWORK_INFO_PROVIDER = "blockfrost";
+        REWARDS_PROVIDER = "blockfrost";
+        UTXO_PROVIDER = "blockfrost";
+        BLOCKFROST_API_KEY = {
+          valueFrom.secretKeyRef = {
+            name = "blockfrost";
+            key = "api-key";
+          };
```

Comment:
Do we have a single token which is valid for all the networks, or should this be improved a bit to pick the correct token based on `config.network`?